app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -698,6 +746,12 @@ class PluginApi {
    * ```
    **/
   addPosterIcons(cb) {
+    // TODO (glimmer-post-stream) what should replace this API?
```

Comment:
I think we should aim to keep this API working. The interface is not widget-based, so it should be very feasible. Themes/plugins just pass in icon/emoji/className/url/title/text.

Reply:
The API is now compatible with the Glimmer Post Component